src/base/bittorrent/session.cpp

```diff
@@ -3703,6 +3729,32 @@ void Session::setMultiConnectionsPerIpEnabled(const bool enabled)
     configureDeferred();
 }
 
+bool Session::validateHTTPSTrackers() const
+{
+    return m_validateHTTPSTrackers;
+}
+
+void Session::setValidateHTTPSTrackers(const bool enabled)
+{
+  if (enabled != m_includeOverheadInLimits) {
+      m_validateHTTPSTrackers = enabled;
+      configureDeferred();
+  }
+}
+
+SSLVersion Session::sslVersion() const
+{
+    return m_sslVersion;
+}
+
+void Session::setSSLVersion(const SSLVersion version)
+{
+    if (version == m_sslVersion) return;
+
+    m_sslVersion = version;
+    configureDeferred();
+}
```

Comment:
Don't you need the include guards here as well? The input value might be unsupported. Or at least select TLSv1.2 instead, in case TLSv1.3 is not supported.

Reply:
With the way I was handling it above, TLS 1.3 wouldn't be selectable if not an option (in the GUI, at least)

Reply:
Oh, yeah, I guess I had the WebUI in mind when I wrote this.